### DISCUSSION TITLE
Looking to create a Fork to use for a different gameline, any advice on getting started?

### DISCUSSION THREAD

**Discussion by CorbinBogert** [naturalcrit/homebrewery#1259] [Q&A] [answered]
Recently had a discussion with a couple of Chronicles of Darkness groups as to how useful this would be for the Storyteller's Vault on Drive Thru RPG. 

If I wanted to fork **to use this code for another product what would I need?** And **which files would I need to likely modify?**

I know this can be a big ask, but I need a jumping off point to start from.

The files I have access to are the following:
https://www.storytellersvault.com/product/259726/Chronicles-of-Darkness-Mortals-Templates
(As well as other variants based on which gameline it is e.g. Red for Vampires)

It has Word and InDesign files particular for InDesign:
- .idml file
- .indd file
- Font files (.lst, ttf)

For Word:
- .docx files
- Font files (lst, tff, otf)

Again this will likely be a big undertaking, but I'm interested in trying to get it done.

**Comment by calculuschild** [comment, marked as answer]
Depending what you want to do, you can simply modify the CSS on the page to change the appearance of your document to match another style. For example, we have a user on Reddit @Gazook who often makes [custom styles](https://www.reddit.com/r/homebrewery/comments/leeju9/theme_add_2e_monstrous_compendium_v1/) to match older versions of D&D. It may be as simple as creating your custom style and supplying that style in a file to your other users.

If you want to fork the project, you can take a look at the readme.md file on this github for download/install instructions. The styles themselves are inside a file called ".phb.less" which you can find somewhere in the "client/" folder if you dig around a bit.

**Comment by Gazook89** [comment]
Hey @CorbinBogert, I was just randomly flipping through this repository and the discussions and found this one, and that calculuschild mentioned myself and my willingness to make themes.  If you are still looking for an option, let me know.  I think just making a theme to be used in Homebrewery is the best way to go rather than trying to reinvent the wheel/forking the whole project.  

I looked at the link you provided with the Word/InDesign templates and while useful for individual graphics and fonts, I would still need to see a handful of actual pages of the book so I can see how it all goes together.  I used to be able to find such books on a certain archive that seems to have disappeared from the internet recently...

You can reach me either here, on reddit as Gazook89, or on discord LongJohnJones#2524
